src/electron/adapters/electron-app-data-adapter.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { app } from 'electron';
+import { AppDataAdapter } from '../../common/browser-adapters/app-data-adapter';
+
+export class ElectronAppDataAdapter implements AppDataAdapter {
+    public getVersion(): string {
+        return app.getVersion();
```

Comment:
what's the value we're actually getting here while running the app? is the value we have on `package.json`. I'm not sure we actually want to use that value or something else. I guess my concern is going to be resolve a little down the path, so this doesn't need to be resolve in this PR

Reply:
The value that we get from this is the electron version. https://electronjs.org/docs/api/app#appgetversion

Reply:
We are not actually changing the version on package.json (it's been 1.0.4 since before we move to git hub).
I think this is fine for now, I'm assuming we're going to have an electron app release feature where we actually take care of the app version.